src/setup/index.ts

```diff
@@ -8,6 +9,13 @@ import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import platformSetup from './platformSetup';
 
+function initializeLastVisitedPath(): string | undefined {
+    if (!(getPlatform() === CONST.PLATFORM.ANDROID || getPlatform() === CONST.PLATFORM.IOS)) {
```

Comment:
@HezekielT please don't use platform specific code. Please use platform specific files for this function.
Nothing wrong with this but we want to be consistent with codestyle